test/e2e/rules_test.go

```diff
@@ -12,6 +12,65 @@ import (
 	"github.com/efficientgo/tools/core/pkg/testutil"
 )
 
+const recordingRuleYamlTpl = `
```

Comment:
I think to make the e2e tests pass we need now to update this const as in [here](https://github.com/observatorium/api/blob/main/test/e2e/configs.go#L205-L212) 
after that LGTM :)

Reply:
Whoops, missed this during merge conflict, thank you :bow: